addons/account/views/partner_view.xml

```diff
@@ -112,8 +112,10 @@
                             </field>
                         </group>
                         </page>
+                        <page name="note" string="Notes">
+                            <field name="note" placeholder="Legal Notes..."/>
+                        </page>
```

Comment:
Is there any particular reason for this change in behavior?
The old behavior, with the field always being at the bottom of the form, was what we expected.

Reply:
I though it was a mistake because nothing shown that the notes section was not part of a tabs.
So I  put it in a separated pad as it's common places in other modules.

But if it's not desired, I'll revert this change.

Reply:
As far as I know, it is there that way on purpose indeed.
I asked the question to our PO and for him, it should not change.
Thank you!